makeDarwinImage/run.nix

```diff
@@ -45,9 +46,18 @@ writeShellScriptBin "run-macOS.sh" ''
     "$@"
   )
 
+  if [ ! -L macos-ventura-base-image.qcow2 ]; then
+  ${if createDiskImageIfNotExists then ''
+    echo 'The disk image macos-ventura-base-image.qcow2 was not found, creating it:'
+    nix-store --realise ${diskImage} --add-root macos-ventura-base-image.qcow2
+  '' else ''
+    echo 'The disk image macos-ventura-base-image.qcow2 was not found but we don'"'"'t have the ability to create it. Please enable the services.macos-ventura.createDiskImageIfNotExists option and try again.'
```

Comment:
`don'"'"'t`

Can you just say "do not" instead?

Reply:
Sure.